### ISSUE TITLE
Getting configuration file working on Windows

### ISSUE THREAD

**Issue by ppatierno** [strimzi/strimzi-kafka-bridge#202]
The current way to get the configuration file should not work on Windows due to the way the absolute paths starts (i.e. C:\).

**Comment by stanlyDoge** [comment]
It works on Windows.

**Comment by scholzj** [comment]
So should it be closed?

**Comment by stanlyDoge** [comment]
Yes, I believe. I tried to run bridge from the IDE and the path was created correctly & the configuration was loaded successfully.

**Comment by scholzj** [comment]
And if you try it outside of IDE with absolute path?
